src/env.ts

```diff
@@ -0,0 +1,16 @@
+import dotenv from 'dotenv';
```

Comment:
The point of this file is to do validation at server start, since that gives the student more immediate feedback

Reply:
Great idea! I like this quite a lot. I'll have to do something similar with my Discord bot.